rocketpy/AeroSurface.py

```diff
@@ -1892,53 +1885,113 @@ def allInfo(self):
         return None
 
 
-class RailButtons:
+class RailButtons(AeroSurface):
```

Comment:
Since the class represents a pair of RailButtons, it is fine to have its name pluralized.